src/Api/Endpoints/Magic.cs

```diff
@@ -26,12 +27,20 @@ public static class MagicEndpoints
     /// <summary>
     /// Adds a rate limiter policy for the MagicEndpoints. Each tenant will have its own partition.
     /// </summary>
-    /// <param name="builder">The RateLimiterOptions builder.</param>
     public static void AddMagicRateLimiterPolicy(this RateLimiterOptions builder) =>
         builder.AddPolicy(RateLimiterPolicy, context =>
         {
             var tenant = context.User.FindFirstValue(CustomClaimTypes.AccountName) ?? "<global>";
 
+            var isRateLimitBypassed = context.RequestServices
+                .GetRequiredService<IConfiguration>()
+                .GetSection("ApplicationOverrides")
+                .TryGetApplicationOverrides(tenant)
+                ?.IsRateLimitBypassEnabled == true;
```

Comment:
Instead of a nullable, we could just have it return a default implementation that is false.